content/chainguard/chainguard-images/about/versions.md

```diff
@@ -58,6 +58,35 @@ The table provides some example scenarios to help illustrate our approach.
 
 > **Note**: The "Maintained Upstream Releases" column is current as of December 2024.
 
+
+## EOL Grace Period
+
+There are cases where an organization may want to continue using an image after it has reached end of life (EOL). This could be because an image reaches EOL before the organization's release schedule, or perhaps later image versions have one or more issues that prevent the organization from upgrading.
+
+To help in situations like this, Chainguard offers an end-of-life grace period for eligible images. By purchasing a grace period, customers can retain access to versions of images whose primary package has entered its end-of-life phase for up to six months after they have reached EOL.
```

Comment:
You're probably going to say something about adjectives and nouns, but note you've used -s here but not before.

Reply:
"retain access" is maybe not quite right. Customers never lose access to images, they just don't get access to new images. Maybe "customers will get access to new builds of images" or something?